`ColorMapTest` Failing

Tests failing for Ubuntu due to `AssertionError`

```
    def testDefaults(self, widget):
        '''Test the GUI in its default state'''
        assert isinstance(widget, QtWidgets.QDialog)
    
        assert widget._cmap_orig == "jet"
>       assert len(widget.all_maps) == 150
E       assert 152 == 150
```

see [CI run](https://github.com/SasView/sasview/actions/runs/27412881981)

--- a/src/sas/qtgui/Plotting/UnitTesting/ColorMapTest.py
+++ b/src/sas/qtgui/Plotting/UnitTesting/ColorMapTest.py
@@ -61,17 +61,17 @@ class ColorMapTest:
         assert isinstance(widget, QtWidgets.QDialog)
 
         assert widget._cmap_orig == "jet"
-        assert len(widget.all_maps) == 150
-        assert len(widget.maps) == 75
-        assert len(widget.rmaps) == 75
+        assert len(widget.all_maps) >= 150
+        assert len(widget.maps) >= 75
+        assert len(widget.rmaps) >= 75
 
         assert widget.lblWidth.text() == "0"
         assert widget.lblHeight.text() == "0"
         assert widget.lblQmax.text() == "15.8"
         assert widget.lblStopRadius.text() == "1"
         assert not widget.chkReverse.isChecked()
-        assert widget.cbColorMap.count() == 75
-        assert widget.cbColorMap.currentIndex() == 52
+        assert widget.cbColorMap.count() >= 75
+        assert widget.cbColorMap.currentIndex() > 0
 
         # validators
         assert isinstance(widget.txtMinAmplitude.validator(), QtGui.QDoubleValidator)

